### ISSUE TITLE
`d` key not working in Ubuntu 22.04

### ISSUE THREAD

**Issue by beatonma** [Leleat/Tiling-Assistant#200]
This is an odd one!

After upgrading Ubuntu (20.04 to 22.04.1), enabling this extension disables my `d` key. *Something* happens when i hit that key but no character output in any application. There is a short delay (a few ms) before any other key is able to register.

Using `shift-d` (`D`) doesn't work either; although `ctrl-d`, `alt-d`, and `super-d` do work as hotkeys in various apps.

All other keys work fine.

https://user-images.githubusercontent.com/12682046/185195154-e59f901e-1020-476c-8971-06bf2d04db22.mp4

I thought it might be a hotkey conflict but it still happens after disabling all of the hotkeys in the extension settings. Restarting the system doesn't have any effect.

I wouldn't be surprised if this was an unreproducible quirk of my system but it's definitely a new quirk which only happens when the extension is enabled so thought it worth reporting.

**System Info:**
 - Distro: Ubuntu 22.04.1
 - GNOME: 42.2
 - Extension version: EGO 35
 - XOrg

**Journalctl logs** No log events.

**Comment by Leleat** [comment]
> This is an odd one!

This is... indeed very odd. Unfortunately, I don't have access to my linux system for 2 weeks, so there isn't much I can investigate myself yet. So...

- Did you only update your distro or did you also update/change anything with T-A? 
- Can you reset all of T-A settings? I don't know the command off the top of my head but you could use `dconf-editor` for the GUI way. The path to the settings is `/org/gnome/shell/extensions` (I believe). Then right-click T-A's settings and 'reset recursively'. Restart GNOME Shell and see if it fixes it.
- Does re-installing or using an older extension version fix it? (Don't forget to reload GNOME Shell after re-installing T-A)

**Comment by beatonma** [comment]
> The path to the settings is `/org/gnome/shell/extensions`

![tiling-assistant-fix](https://user-images.githubusercontent.com/12682046/185363198-ae3eecd3-c0f4-4d9a-bb63-22ba05c605e8.png)

Aha! I had `activate-layout0` set to `d` somehow - I have no idea how or when I did that but removing it fixed the issue.

ddddd

Thanks for your help! Also thank you for the extension in general - I didn't realise how often I used it until I had to disable it for a few hours.
